src/inet/InetInterface.cpp

```diff
@@ -84,7 +84,7 @@ CHIP_ERROR InterfaceId::GetInterfaceName(char * nameBuf, size_t nameBufSize) con
 }
 CHIP_ERROR InterfaceId::InterfaceNameToId(const char * intfName, InterfaceId & interface)
 {
-    if (strlen(intfName) < 3)
+    if (strnlen(intfName, kMaxIfNameLength) < 3)
```

Comment:
This change ... does not seem to make anything safer.  If there is no null-terminator before `kMaxIfNameLength` then nothing would change about this test and then the `strtoul` below will happily walk off the end of the data just like `strlen` would have.  If there _is_ a null-terminator before `kMaxIfNameLength`, then `strlen` would not be a problem either.